StoichiometryFitter.py

```diff
@@ -567,23 +567,12 @@ def OnOpen(self, event):  # wxGlade: MyFrame.<event_handler>
             S = genfromtxt(dlg.GetPath(), skip_header=5, skip_footer=2, dtype=None)
             # Drop it into pandas and sort based on elemental Z.
             pdS = pd.DataFrame(S).sort_values('f1')
-            BrukerInput = pdS.to_numpy()[:, (0, 3)].T
+            BrukerInput = pdS.as_matrix()[:, (0, 3)].T
```

Comment:
I had made modifications to stoichiometryfitter.py.  Before issuing a pull request you need to git pull the remote origin in order to get any changes other authors have done.  Here you pushed an old version (as_matrix doesn't work in python 3, and the utf-8 decoding is needed in python 3).  I suggest you revert stoichiometryfitter.py and then issue a new pull request.  BTW, this is one reason to keep commits small.  Now the whole commit is held up because of this little thing.  :-)